src/main/kotlin/com/vk/admstorm/AdmStartupService.kt

```diff
@@ -122,6 +126,21 @@ class AdmStartupService(private var project: Project) {
         })
     }
 
+    private fun setupIssueTrackers(project: Project) {
+        val navigationConfig = IssueNavigationConfiguration.getInstance(project)
+        val service = Env.data.services.getByKey("jira") ?: return
+        val url = service.url + "/browse/\$0"
+
+        val isExist = navigationConfig.links.any { it ->
```

Comment:
`it` is not needed. The condition can be written in one line

Reply:
I prefer to always specify lambdas explicitly, but if it's important, then ok